### PR TITLE
[IMP] inventory: sendcloud shipping rules

### PR DESCRIPTION
**Pull request by Felicious**
Add section about adding Sendcloud shipping rules and necessary Odoo configuration.

Fwport: yes

[Task](https://www.odoo.com/odoo/project/3835/tasks/4033574?cids=3)

### CODE REVIEW

**Comment by Felicious** [comment]
Hi @larm-odoo ! This very quick PR is ready for review (:

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst [hunk 1]
```diff
@@ -217,6 +217,34 @@ Additionally, the tracking number is now available.
...
+Optionally, create shipping rules to automatically generate shipping labels tailored to different
+products' needs. For example, a shipping rule can be created for customers shipping expensive
+jewelry items to purchase insurance.
+
+To use shipping rules, navigate to :menuselection:`Inventory app --> Configuration --> Shipping
+Methods` and select the intended `Sendcloud` shipping method.
+
+In the :guilabel:`Sendcloud Configuration` tab, in the :guilabel:`Use Sendcloud shipping rules`
+field, choose the kind of shipments the shipping rules apply to: :guilabel:`Shipping` to customers,
+:guilabel:`Returns`, or :guilabel:`Both`.
```
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst:232]:
```suggestion
In the :guilabel:`OPTIONS` section of the :guilabel:`Sendcloud Configuration` tab,  in the :guilabel:`Use Sendcloud shipping rules`
field, choose the kind of shipments the shipping rules apply to: :guilabel:`Shipping` to customers,
:guilabel:`Returns` from cusotmers, or :guilabel:`Both`.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst [hunk 2]
```diff
@@ -217,6 +217,34 @@ Additionally, the tracking number is now available.
...
+
+.. image:: sendcloud_shipping/enable-shipping-rules.png
+   :align: center
+   :alt: Use Shipping Rules field.
+
+Then, in the Sendcloud website, navigate to :menuselection:`Settings --> Shipping rules`. Create a
+new shipping rule by clicking :guilabel:`Create New`.
+
+Set a :guilabel:`Condition` to determine when the rule applies, in the :guilabel:`Actions` section,
+configure what to do when packages meet the condition.
```
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst:242]:
```suggestion
Set a :guilabel:`Condition` to determine when the rule applies, in the :guilabel:`Actions` section, then
configure what to do when packages meet the condition.
```

**Review by larm-odoo — APPROVED** [review]
Short and sweet! All set for you!

**Comment by Felicious** [comment]
Hii @odoo/us-doc-review ! This short PR is ready for your review (:

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst [hunk 3]
```diff
@@ -217,6 +217,34 @@ Additionally, the tracking number is now available.
    When return labels are created, Sendcloud will automatically charge the configured Sendcloud
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst:217]:
```suggestion
   When return labels are created, Sendcloud automatically charges the configured Sendcloud
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst [hunk 4]
```diff
@@ -217,6 +217,34 @@ Additionally, the tracking number is now available.
...
    account.
 
+Shipping rules
+--------------
+
+Optionally, create shipping rules to automatically generate shipping labels tailored to different
+products' needs. For example, a shipping rule can be created for customers shipping expensive
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst:224]:
```suggestion
product needs. For example, a shipping rule can be created for customers shipping expensive
```
just sounds a bit cleaner this way -- totally optional, though. completely your call. 👍

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst [hunk 5]
```diff
@@ -217,6 +217,34 @@ Additionally, the tracking number is now available.
...
+jewelry items to purchase insurance.
+
+To use shipping rules, navigate to :menuselection:`Inventory app --> Configuration --> Shipping
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst:227]:
```suggestion
To use shipping rules, navigate to :menuselection:`Inventory app --> Configuration --> Delivery: Shipping
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst [hunk 6]
```diff
@@ -217,6 +217,34 @@ Additionally, the tracking number is now available.
...
+Methods` and select the intended `Sendcloud` shipping method.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst:228]:
```suggestion
Methods`, and select the intended `Sendcloud` shipping method.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst [hunk 7]
```diff
@@ -217,6 +217,34 @@ Additionally, the tracking number is now available.
...
+
+In the :guilabel:`OPTIONS` section of the :guilabel:`Sendcloud Configuration` tab,  in the :guilabel:`Use Sendcloud shipping rules`
+field, choose the kind of shipments the shipping rules apply to: :guilabel:`Shipping` to customers,
+:guilabel:`Returns` from customers, or :guilabel:`Both`.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst:232]:
```suggestion
Under the :guilabel:`Sendcloud Configuration` tab, in the :guilabel:`OPTIONS` section, choose the kind
of shipments the shipping rules apply to, via the :guilabel:`Use Sendcloud shipping rules` field.

From here, choose either: :guilabel:`Shipping` to customers, :guilabel:`Returns` from customers, or :guilabel:`Both`.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst [hunk 8]
```diff
@@ -217,6 +217,34 @@ Additionally, the tracking number is now available.
...
+
+.. image:: sendcloud_shipping/enable-shipping-rules.png
+   :align: center
+   :alt: Use Shipping Rules field.
+
+Then, in the Sendcloud website, navigate to :menuselection:`Settings --> Shipping rules`. Create a
+new shipping rule by clicking :guilabel:`Create New`.
+
+Set a :guilabel:`Condition` to determine when the rule applies, in the :guilabel:`Actions` section, then
+configure what to do when packages meet the condition.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst:242]:
```suggestion
In the :guilabel:`Actions` section, set a :guilabel:`Condition` to determine when the rule applies. Then,
configure what to do when packages meet the condition.
```

**Review by ksc-odoo — APPROVED** [review]
Hey @Felicious -- just finished my Final Review of this PR! Nice job. Only a small handful of comments that require your attention. But, once you implement the necessary adjustments, feel free to tag this for Tech Review. Thanks! 👍

**Comment by Felicious** [comment]
Hi @samueljlieber ! Welcome back from vacation :blush: This PR is ready for tech review

**Review by samueljlieber — APPROVED** [review]
Looks good to me @Felicious! Nice work, thank you!
..
@robodoo r+

**Pull request closed without merging**